node/src/proxy_server/mod.rs

```diff
@@ -1380,7 +1380,7 @@ mod tests {
     use crate::test_utils::recorder::make_recorder;
     use crate::test_utils::recorder::peer_actors_builder;
     use crate::test_utils::recorder::Recorder;
-    use crate::test_utils::recorder_stop_conditions::{StopCondition, StopConditions};
+    use crate::test_utils::recorder_stop_conditions::{MsgIdentification, StopConditions};
```

Comment:
Inside the macro `match_lazily_every_type_id!()` you would like to put the full import for `MsgIdentification`, then there won't be any need for importing this wherever the macro is being used.